examples/lab-robust/v2/qwiklabs.yaml

```diff
@@ -11,32 +13,14 @@ tags:
 
 resources:
   - type: file
-    title:
-      locales:
-        en: Sample PDF
-        es: Ejemplo de PDF
-    description:
-      locales:
-        en: This PDF contains all of the code samples for the lab.
-        es: Este PDF contiene todos los ejemplos de código para el laboratorio.
-    uri:
-      locales:
-        en: ./resources/sample-en.pdf
-        es: ./resources/sample-es.pdf
+    title: Sample PDF
+    description: This PDF contains all of the code samples for the lab.
+    uri: resources/sample-en.pdf
   - type: video
     id: intro-video
-    title:
-      locales:
-        en: Welcome to GCP!
-        es: ¡Bienvenido a GCP!
-    uri:
-      locales:
-        en: https://www.youtu.be/oHg5SJYRHA0
-        es: https://www.youtu.be/7jjoyy7_RCk
-    description:
-      locales:
-        en: Overview of Google Cloud Platform
-        es: Descripción general de Google Cloud Platform
+    title: Welcome to GCP!
+    uri: https://www.youtu.be/oHg5SJYRHA0
+    description: Overview of Google Cloud Platform
     duration: 360
 
 environment:
```

Comment:
In `environment/resources` needs a couple things
- in gcp_project section : `ssh_key_user: primary_user`
- in gcp_project section : 
```
allowed_locations:
- us-central1-a
```